sdv/datasets/demo.py

```diff
@@ -67,7 +67,9 @@ def _extract_data(bytes_io, output_folder_name):
         if output_folder_name:
             os.makedirs(output_folder_name, exist_ok=True)
             zf.extractall(output_folder_name)
-            os.remove(os.path.join(output_folder_name, 'metadata_v0.json'))
+            metadata_v1_filepath = os.path.join(output_folder_name, 'metadata_v0.json')
```

Comment:
why is it called metadata_v1_filepath but the suffix has v0?

Reply:
Mistake. Fixed https://github.com/sdv-dev/SDV/pull/1865/commits/04d2083ea057aa9fd10f469b0640dc0d481808f4